examples/pos-mainchain/config/default/genesis_assets_with_preset.json

```diff
@@ -0,0 +1,3074 @@
+{
+	"assets": [
+		{
+			"module": "interoperability",
+			"data": {
+				"outboxRootSubstore": [
+					{
+						"storeKey": "04000000",
+						"storeValue": {
+							"root": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+						}
+					},
+					{
+						"storeKey": "04000001",
+						"storeValue": {
+							"root": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+						}
+					}
+				],
+				"chainDataSubstore": [
+					{
+						"storeKey": "04000000",
```

Comment:
same as comment below, there should be no chain account for the mainchain itself

Reply:
Same here. From the code, if I want to suppress the error, the data of mainchain need to present at `chainDataSubstore`